crates/network-scanner-net/src/test.rs

```diff
@@ -179,50 +175,28 @@ async fn work_with_tokio_tcp() -> anyhow::Result<()> {
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
 pub async fn drop_runtime() -> anyhow::Result<()> {
-    let kill_server = Arc::new(AtomicBool::new(false));
-    let (addr, handle) = local_tcp_server(kill_server.clone()).await?;
-    tracing_subscriber::fmt::SubscriberBuilder::default()
-        .with_max_level(tracing::Level::TRACE)
-        .with_thread_names(true)
-        .init();
     {
         let async_runtime = crate::runtime::Socket2Runtime::new(None)?;
         {
-            let good_socket = async_runtime.new_socket(socket2::Domain::IPV4, socket2::Type::STREAM, None)?;
             let bad_socket = async_runtime.new_socket(socket2::Domain::IPV4, socket2::Type::STREAM, None)?;
 
-            tracing::info!("good_socket: {:?}", good_socket);
             tracing::info!("bad_socket: {:?}", bad_socket);
 
             let unused_port = 12345;
 
-            let available_addr = addr;
             let non_available_addr = SocketAddr::from(([127, 0, 0, 1], unused_port));
 
-            let handle =
```

Comment:
Maybe should add this back?

Reply:
If we are already testing the same thing elsewhere, it’s okay to not I guess? As you prefer!